docs/extend/plugins_volume.md

```diff
@@ -140,15 +162,15 @@ at the first mount request and deprovision at the last corresponding unmount req
 **Response**:
 ```json
 {
-    "Mountpoint": "/path/to/directory/on/host",
+    "Mountpoint": "/path/to/directory/inside/plugin",
```

Comment:
This path is essentially `PropagatedMount` property of the config. https://github.com/docker/docker/blob/696ef7a19eccd44732e9c50902c8949e574ef097/docs/extend/config.md

PropagatedMount is needed so that the mounts performed by the volume plugin are visible to the docker daemon, and henceforth can be bind mounted to the containers.